ingress/controllers/nginx-alpha-ssl/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+export VAULT_SKIP_VERIFY=true
+[ -f /etc/vault-token/ingress-read-only ] && export VAULT_TOKEN=`cat /etc/vault-token/ingress-read-only`
```

Comment:
Why not just read this directly in the controller (--vault-token) instead of using an env var? 
If you add it as a secret it will be mounted in the specified location (or it's a bug in Kubernetes).